### PR TITLE
Autoscale: Fix ThroughputProperties factory names

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Manual is being used as the wording for describing the normal throughput offering. This PR is replacing all the places throughput was using fixed to manual. 
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Resource/Settings/ThroughputProperties.cs [hunk 1]
```diff
@@ -91,39 +91,39 @@ private ThroughputProperties(OfferContentProperties offerContentProperties)
...
-        /// <param name="maxAutoscaleThroughput">The staring maximum throughput the resource can scale to.</param>
+        /// <param name="maxAutoscaleThroughput">The maximum throughput the resource can scale to.</param>
         /// <returns>Returns a ThroughputProperties for autoscale provisioned throughput</returns>
 #if PREVIEW
         public
 #else
         internal
 #endif
-        static ThroughputProperties CreateAutoscaleProvionedThroughput(
+        static ThroughputProperties CreateAutoscaleThroughput(
```
Comment (abhijitpai) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/ThroughputProperties.cs:118]: Shouldn't these be Autopilot rather than Autoscale?

Comment (FabianMeiswinkel) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/ThroughputProperties.cs:118]: No - AutoScale is the correct term for GA

Comment (abhijitpai) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/ThroughputProperties.cs:118]: Thanks for confirming.

**Comment by kirankumarkolli** [comment]
Create prefix for factory methods sees like ambiguous on it it really creating or just OM.
Any references on how FX patterns are for siimlar?

**Pull request merged**